src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -137,10 +137,16 @@ try
 {
     const auto drawingContext = static_cast<const DrawingContext*>(clientDrawingContext);
 
-    const DWRITE_FONT_WEIGHT weight = _fontRenderData->DefaultFontWeight();
+    DWRITE_FONT_WEIGHT weight = _fontRenderData->DefaultFontWeight();
     DWRITE_FONT_STYLE style = _fontRenderData->DefaultFontStyle();
     const DWRITE_FONT_STRETCH stretch = _fontRenderData->DefaultFontStretch();
 
+    if (drawingContext->useBoldFont)
+    {
+        // TODO: "relative" bold?

```

Comment:
Do we need to do this? I don't quite know. This was brought up in other discussions that I can't find at the moment. probably somewhere in #109. Basically what we gonna do when use choose "semi-bold" or "bold" as their default font? Do we choose a relative "bolder" font here?

Reply:
https://github.com/microsoft/terminal/issues/109#issuecomment-743929268 is probably the comment you're looking for?

Reply:
Yea I believe so. Thanks.

Here's a joke that I just thought of:

> When they go normal, we go bold.
> When they go bold, we go extra-bold.
>
>  *Michelle "Terminal" Obama*

Reply:
Just multiply by 1.75, cap it at 1000, and call it a day, perhaps. I run font weight 450 locally ("Retina" weight) rather than 400, so this would come out closer to Extra Bold than Bold, which seems reasonable to me. If the font doesn't support variable-width, let DirectWrite work out which weight instance to pick.

If it's helpful, CSS defines [its own transformation for bold](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#meaning_of_relative_weights), but that doesn't allow for variable-weight fonts.

Reply:
This is useful information. Thanks. And I believe DirectWrite can work out which weight instance to pick, even under those seemingly invalid situations.

Reply:
I'm not sure what you mean by invalid. From what I can see, the API will reject invalid weights (<1, >999), and any other weight is valid, although if you're not using a variable-width font, DWrite must pick the nearest matching font variant.

Reply:
```suggestion
        // TODO:GH#10577 "relative" bold?
```